### PR TITLE
Updating Docs to latest tbdex-kt

### PR DESCRIPTION
**Pull request by acekyd**
## What type of PR is this? (check all applicable)

- [x] ♻️ Refactor
- [x] 📝 Documentation Update
- [x] 🧪 Tests

## Description

This PR updates tbdex to v0.11.0-beta and its transitive dependencies. Also updates existing tests to match.

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?
- [x] 📓 general documentation

## [optional] What gif best describes this PR or how it makes you feel?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206820426580956

### CODE REVIEW

**Comment by blackgirlbytes** [comment]
I think for Verify VC it's failing because Did Dht doesn't have certain verification methods for signing yet..but did jwk worked when I tried to write it out in my local env on intellij

here's a messy example..gonna try it on your branch now

```kotlin

import com.fasterxml.jackson.databind.ObjectMapper
import web5.sdk.credentials.PresentationExchange
import web5.sdk.credentials.VerifiableCredential
import web5.sdk.credentials.VerifiablePresentation
import web5.sdk.credentials.model.ConstraintsV2
import web5.sdk.credentials.model.FieldV2
import web5.sdk.credentials.model.InputDescriptorV2
import web5.sdk.credentials.model.PresentationDefinitionV2
import web5.sdk.crypto.InMemoryKeyManager
import web5.sdk.dids.methods.dht.DidDht
import web5.sdk.dids.methods.jwk.DidJwk
import web5.sdk.dids.methods.key.DidKey

data class VerificationResult(
    val jwt: String,
    val result: Any?, // Replace Any with the actual type of your verification result
    val isValid: Boolean,
    val error: String?
)

fun main() {
    val issuerDid = DidKey.create(InMemoryKeyManager())
    val aliceDid = DidKey.create(InMemoryKeyManager())
    val youDht = DidDht.create(InMemoryKeyManager())
    val newIssuerDid = DidJwk.create(InMemoryKeyManager())
    val newAliceDid = DidJwk.create(InMemoryKeyManager())

    //    println(youDht.didDocument?.verificationMethods)
//    val resolvedMyDid = DidResolvers.resolve(youDht.uri)
//    println(resolvedMyDid)
//    println("this is the DID’s public key information  ${youDht.didDocument?.verificationMethods}")
//    // Define data classes for each credential
    data class EmploymentCredential(
        val employmentStatus: String
    )

    //
    data class DateOfBirthCredential(
        val dateOfBirth: String
    )

    //
    data class NameCredential(
        val name: String
    )

    data class NameAndDobCredential(
        val name: String,
        val dateOfBirth: String
    )
//
//    // Create instances of credentials
    val employmentVc = VerifiableCredential.create(
        type = "EmploymentCredential",
        issuer = newIssuerDid.uri,
        subject = newAliceDid.uri,
        data = EmploymentCredential(employmentStatus = "employed")
    )

//    val dobVc = VerifiableCredential.create(
//        type = "DateOfBirthCredential",
//        issuer = issuerDid.uri,
//        subject = aliceDid.uri,
//        data = DateOfBirthCredential(dateOfBirth = "1990-01-01")
//    )
//
//    val nameVc = VerifiableCredential.create(
//        type = "NameCredential",
//        issuer = issuerDid.uri,
//        subject = aliceDid.uri,
//        data = NameCredential(name = "Alice Bob")
//    )
//
    val nameAndDobVc = VerifiableCredential.create(
        type = "NameAndDobCredential",
        issuer = newIssuerDid.uri,
        subject = newAliceDid.uri,
        data = NameAndDobCredential(name = "alice bob", dateOfBirth = "1990-10-01")
    )
//
//    println(nameAndDobVc)
    val signedNameAndDobVc = nameAndDobVc.sign(newIssuerDid)
//    println(signedNameAndDobVc)
//
//    // Sign the credentials
    val signedEmploymentVc = employmentVc.sign(newIssuerDid)
    //  println(signedEmploymentVc)
//    val signedDobVc = dobVc.sign(issuerDid)
//    val signedNameVc = nameVc.sign(issuerDid)
//
//    // Define the presentation definition
    val presentationDefinition = PresentationDefinitionV2(
        id = "presDefIdloanAppVerification123",
        name = "Loan Application Employment Verification",
        purpose = "To verify applicant’s employment, date of birth, and name",
        inputDescriptors = listOf(
            InputDescriptorV2(
                id = "employmentVerification",
                purpose = "Confirm current employment status",
                constraints = ConstraintsV2(
                    fields = listOf(
                        FieldV2(
                            path = listOf("$.vc.credentialSubject.employmentStatus"),
                            filterJson = ObjectMapper().readTree(
                                """
                        {
                          "type": "string",
                          "pattern": "employed"
                        }
                        """
                            )
                        )
                    )
                )
            ),
            InputDescriptorV2(
                id = "dobVerification",
                purpose = "Confirm the applicant's date of birth",
                constraints = ConstraintsV2(
                    fields = listOf(
                        FieldV2(
                            path = listOf("$.vc.credentialSubject.dateOfBirth"),
                            filterJson = ObjectMapper().readTree(
                                """
                        {
                          "type": "string",
                          "format": "date"
                        }
                        """
                            )
                        )
                    )
                )
            ),
            InputDescriptorV2(
                id = "nameVerification",
                purpose = "Confirm the applicant’s legal name",
                constraints = ConstraintsV2(
                    fields = listOf(
                        FieldV2(
                            path = listOf("$.vc.credentialSubject.name"),
                            filterJson = ObjectMapper().readTree(
                                """
                        {
                          "type": "string"
                        }
                        """
                            )
                        )
                    )
                )
            )
        )
    )
    val firstVerification = VerifiableCredential.verify(signedNameAndDobVc)
    println(firstVerification)

    val presentationResult = PresentationExchange.createPresentationFromCredentials(
        vcJwts = listOf(signedEmploymentVc, signedNameAndDobVc),
        presentationDefinition = presentationDefinition
    )
    println(presentationResult)

    val verifiablePresentation = VerifiablePresentation.create(
        vcJwts = listOf(signedEmploymentVc, signedNameAndDobVc),
        holder = newAliceDid.uri,
        additionalData = mapOf("presentation_submission" to presentationResult)
    )
    println(verifiablePresentation)

    val vpDataMap = verifiablePresentation.vpDataModel.toMap()
    val vcJwtArray = vpDataMap["verifiableCredential"] as? List<String> ?: emptyList()

    val verificationResults = mutableListOf<VerificationResult>()
    var errorsFound = false

    vcJwtArray.forEach { vcJwt ->
        println("test about to try verification")
        try {
            val verificationResult = VerifiableCredential.verify(vcJwt)
            println("I am verification result")
            println(verificationResult)
            verificationResults.add(VerificationResult(vcJwt, verificationResult, true, null))
        } catch (e: Exception) {
            errorsFound = true
            verificationResults.add(VerificationResult(vcJwt, null, false, e.message))
        }
    }
}
```

**Comment by blackgirlbytes** [comment]
okay I got two working JWTs on your branch locally..let me know if I could push to your branch or if I should comment with a suggestion that you can commit!

**Comment by acekyd** [comment]
Thanks @blackgirlbytes. DidJwk works - I've started a convo on Slack for more context on why it's the only one that works at the moment. 🙏🏽

**Review by angiejones — APPROVED** [review]

File: site/testsuites/testsuite-kotlin/src/test/kotlin/docs/utils/TestData.kt [hunk 1]
```diff
@@ -113,7 +113,7 @@ object TestData {
     fun getQuote(
         to: String = ALICE_DID.uri,
         from: String = PFI_DID.uri) = Quote.create(
-        ALICE_DID.uri, PFI_DID.uri, TypeId.generate(MessageKind.rfq.name),
+        ALICE_DID.uri, PFI_DID.uri, TypeId.generate(MessageKind.rfq.name).toString(),
```
Comment (chris-tbd) [inline site/testsuites/testsuite-kotlin/src/test/kotlin/docs/utils/TestData.kt:116]: Do we still need to do this `TypeId.generate()` or can we get around this now?

**Review by chris-tbd — APPROVED** [review]

File: site/testsuites/testsuite-kotlin/src/test/kotlin/docs/utils/TestData.kt (hunk 1 above)
Comment (acekyd) [reply site/testsuites/testsuite-kotlin/src/test/kotlin/docs/utils/TestData.kt:116]: I believe so. The examples from the tests in the sdk show this

**Pull request merged**